app/views/shared/_flash.html.erb

```diff
@@ -19,6 +19,12 @@
 
 <script>
   $('.close').click(function(){
-    $('.text-center').remove();
+    $('.flash_msg').remove();
+  });
+
+  $(document).ready(function(){
+    setInterval(function(){
+      $('.close').trigger('click');
```

Comment:
Instead of triggering event, use `$('.flash_msg').remove();`

Also this flash message removal line would be common after this change. so make a new function and call that function from both above function